code/modules/research/xenoarchaeology/artifact/artifact_replicator.dm

```diff
@@ -21,48 +21,105 @@
 	..()
 
 	var/list/viables = list(
+		//Medical
+		/obj/item/weapon/reagent_containers/glass/bottle/chloralhydrate,
 		/obj/item/roller,
-		/obj/structure/closet/crate,
-		/obj/structure/closet/acloset,
+		/obj/item/device/healthanalyzer,
+		/obj/item/weapon/reagent_containers/glass/bottle/chloralhydrate,
+		/obj/item/weapon/reagent_containers/syringe/inaprovaline,
+		/obj/item/weapon/reagent_containers/glass/bottle/capsaicin,
+		/obj/item/weapon/reagent_containers/glass/bottle/frostoil,
+		/obj/item/weapon/reagent_containers/glass/beaker/large/erlenmeyer,
+
+		//Robots (and the mimic)
 		/mob/living/simple_animal/hostile/mimic,
 		/mob/living/simple_animal/hostile/viscerator,
 		/mob/living/simple_animal/hostile/hivebot,
+		/obj/machinery/bot/cleanbot/roomba, //UNLEASH THE HORDE
+		/obj/machinery/bot/secbot, //Probably a bad idea.
+
+		//Tools
 		/obj/item/device/analyzer,
-		/obj/item/device/camera,
 		/obj/item/device/flash,
-		/obj/item/device/flashlight,
-		/obj/item/device/healthanalyzer,
 		/obj/item/device/multitool,
-		/obj/item/device/paicard,
-		/obj/item/device/radio,
-		/obj/item/device/radio/headset,
-		/obj/item/beacon,
-		/obj/item/weapon/autopsy_scanner,
-		/obj/item/weapon/bikehorn,
-		/obj/item/weapon/bonesetter,
-		/obj/item/weapon/kitchen/utensil/knife/large/butch,
-		/obj/item/weapon/caution,
-		/obj/item/weapon/caution/cone,
-		/obj/item/weapon/crowbar,
-		/obj/item/weapon/storage/bag/clipboard,
-		/obj/item/weapon/cell,
-		/obj/item/weapon/circular_saw,
-		/obj/item/weapon/hatchet,
-		/obj/item/weapon/handcuffs,
-		/obj/item/weapon/hemostat,
-		/obj/item/weapon/kitchen/utensil/knife/large,
-		/obj/item/weapon/lighter/random,
-		/obj/item/weapon/light/bulb,
-		/obj/item/weapon/light/tube,
+		/obj/item/device/camera,
+		/obj/item/weapon/crowbar/red,
+		/obj/item/weapon/lighter/zippo,
 		/obj/item/weapon/pickaxe,
 		/obj/item/weapon/pickaxe/shovel,
-		/obj/item/weapon/table_parts,
-		/obj/item/weapon/weldingtool,
-		/obj/item/weapon/wirecutters,
-		/obj/item/weapon/wrench,
-		/obj/item/weapon/screwdriver,
+		/obj/item/stack/cable_coil/random,
+		/obj/item/weapon/soap/,
+		/obj/item/device/mass_spectrometer,
```

Comment:
These are now defunct and should be replaced with reagent scanners.

Reply:
Unless it is intentional. 
These should be removed from the code but still exist just in case they are referred to somewhere.

Reply:
Use `/obj/item/device/reagent_scanner/adv` instead

Reply:
replaced. I added mass spectrometer because theyre niche and unique, but since they could get removed any moment now I'll just swap them with what you suggested